lvsfunc/misc.py

```diff
@@ -415,6 +417,50 @@ def upscale_chroma(n: int, f: vs.VideoFrame) -> vs.VideoNode:
     return cast(F, inner)
 
 
+def colored_clips(amount: int,
+                  max_hue: int = 300,
+                  rand: bool = True,
+                  seed: Optional[Union[bytearray, bytes, float, int, str]] = None,
+                  **kwargs: Any
+                  ) -> List[vs.VideoNode]:
+    """
+    Returns a list of BlankClips with unique colors (equally spaced in the HSL's hue domain) in sequential or random order.
+    Useful maybe for comparison functions or just for getting multiple uniquely colored BlankClips for testing purposes.
+
+    Will always return a pure red clip in the list as this is the RGB equivalent of the lowest HSL hue possible (0).
+
+    Written by Dave <orangechannel@pm.me>.
+
+    :param amount:  Number of ``vapoursynth.VideoNode``s to return
+    :param max_hue: Maximum hue (0 < hue <= 360) in degrees to generate colors from (uses the HSL color model).
+                    Setting this higher than ``315`` will result in the clip colors looping back towards red and is not recommended for visually distinct colors.
+                    If the `amount` of clips is higher than the `max_hue` expect there to be identical or visually similar colored clips returned (Default: 300)
+    :param rand:    Randomizes order of the returned list (Default: True)
+    :param seed:    Bytes-like object passed to ``random.seed`` which allows for consistent randomized order of the clips (Default: None)
+    :param kwargs:  Arguments passed to ``vapoursynth.core.std.BlankClip`` (Default: keep=1)
+
+    :return:        List of uniquely colored clips in sequential or random order.
+    """
+    if amount < 2:
+        raise ValueError("colored_clips: `amount` must be at least 2")
+    if not (0 < max_hue <= 360):
+        raise ValueError("colored_clips: `max_hue` must be greater than 0 and less than 360 degrees")
+
+    blank_clip_args: Dict[str, Any] = {'keep': 1, **kwargs}
+
+    hues: List[Union[float, int]] = [i * max_hue / (amount - 1) for i in range(amount - 1)]
```

Comment:
`/` always returns a float, so there is no need for the union.

Reply:
That's because I append `max_hue` at the end and mypy doesn't like that since it's an int, also its kind of an artifact of originally starting the list with just `0`.